### PR TITLE
Define a constant instead of duplicating "backpacks."

### PR DESCRIPTION
**Pull request by b10n1k**
## Description
I found this small problem in https://sonarcloud.io/project/issues?id=Slimefun_Slimefun4&open=AXTBL6kaIVI46B1gLWcP&resolved=false
I defined a new private variable for it. I am not sure about the name and also i believe that this could also be defined as static in the class.

## Changes
- define private variable `backpacks`
- replace hardcoded string with the new variable

## Related Issues
NA

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
Please don't use labels in your title.

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
     private Inventory inventory;
     private int size;
+    private final static String backpacks = "backpacks.";
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:37]: Wrong modifier order, this should be `private static final`.

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 2]
```diff
@@ -44,10 +45,10 @@
      *            The id of this Backpack
      */
     public PlayerBackpack(@Nonnull PlayerProfile profile, int id) {
-        this(profile, id, profile.getConfig().getInt("backpacks." + id + ".size"));
+        this(profile, id, profile.getConfig().getInt(PlayerBackpack.backpacks + id + ".size"));
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:48]: `PlayerBackpack` is redundant here, no need to specify the class you are already in.

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 3]
```diff
@@ -44,10 +45,10 @@
...
 
         for (int i = 0; i < size; i++) {
-            inventory.setItem(i, cfg.getItem("backpacks." + id + ".contents." + i));
+            inventory.setItem(i, cfg.getItem(PlayerBackpack.backpacks + id + ".contents." + i));
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:51]: `PlayerBackpack` is redundant here, no need to specify the class you are already in.

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 4]
```diff
@@ -71,7 +72,7 @@ public PlayerBackpack(@Nonnull PlayerProfile profile, int id, int size) {
         this.cfg = profile.getConfig();
         this.size = size;
 
-        cfg.setValue("backpacks." + id + ".size", size);
+        cfg.setValue(PlayerBackpack.backpacks + id + ".size", size);
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:75]: `PlayerBackpack` is redundant here, no need to specify the class you are already in.

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 5]
```diff
@@ -142,7 +143,7 @@ public void setSize(int size) {
         }
 
         this.size = size;
-        cfg.setValue("backpacks." + id + ".size", size);
+        cfg.setValue(PlayerBackpack.backpacks + id + ".size", size);
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:146]: `PlayerBackpack` is redundant here, no need to specify the class you are already in.

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 6]
```diff
@@ -160,7 +161,7 @@ public void setSize(int size) {
      */
     public void save() {
         for (int i = 0; i < size; i++) {
-            cfg.setValue("backpacks." + id + ".contents." + i, inventory.getItem(i));
+            cfg.setValue(PlayerBackpack.backpacks + id + ".contents." + i, inventory.getItem(i));
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:164]: `PlayerBackpack` is redundant here, no need to specify the class you are already in.

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 1 above)
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:37]: Constants should follow the snake-case convention and should be put on top.
Also, this constant's name is a bit meaningless towards what it does, something like `CONFIG_PREFIX` would be more accurate in my opinion.

Here's my suggested change, could you please also move it to the top of the class?

```suggestion
    private static final String CONFIG_PREFIX = "backpacks.";
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
Thank you for your contribution :)
I marked some things that could be improved to follow some conventions

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 1 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: ff83a0183

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 6 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: ff83a0183

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 4 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: ff83a0183

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 3 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: ff83a0183

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 2 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: ff83a0183

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 1 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: ff83a0183

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 7]
```diff
@@ -44,10 +45,10 @@
      *            The id of this Backpack
      */
     public PlayerBackpack(@Nonnull PlayerProfile profile, int id) {
-        this(profile, id, profile.getConfig().getInt("backpacks." + id + ".size"));
+        this(profile, id, profile.getConfig().getInt( CONFIG_PREFIX+ id + ".size"));
```
Comment (poma123) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:48]: Please place that space after the variable.
```suggestion
        this(profile, id, profile.getConfig().getInt(CONFIG_PREFIX + id + ".size"));
```

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 8]
```diff
@@ -44,10 +45,10 @@
...
 
         for (int i = 0; i < size; i++) {
-            inventory.setItem(i, cfg.getItem("backpacks." + id + ".contents." + i));
+            inventory.setItem(i, cfg.getItem( CONFIG_PREFIX+ id + ".contents." + i));
```
Comment (poma123) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:51]: Please place that space after the variable.
```suggestion
            inventory.setItem(i, cfg.getItem(CONFIG_PREFIX + id + ".contents." + i));
```

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 9]
```diff
@@ -71,7 +72,7 @@ public PlayerBackpack(@Nonnull PlayerProfile profile, int id, int size) {
         this.cfg = profile.getConfig();
         this.size = size;
 
-        cfg.setValue("backpacks." + id + ".size", size);
+        cfg.setValue( CONFIG_PREFIX+ id + ".size", size);
```
Comment (poma123) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java:75]: Please place that space after the variable.
```suggestion
        cfg.setValue(CONFIG_PREFIX + id + ".size", size);
```

**Review by poma123 — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 9 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: 64c639ef8

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 8 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: 64c639ef8

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java (hunk 7 above)
Comment (b10n1k) [reply src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: 64c639ef8

**Review by poma123 — APPROVED** [review]
LGTM, thanks for your contribution!

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 10]
```diff
@@ -33,7 +33,7 @@
     private final Config cfg;
 
     private Inventory inventory;
-    private int size;
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: You accidentally removed the `size` here 😬

File: src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java [hunk 11]
```diff
@@ -33,7 +33,7 @@
...
+    private static final String CONFIG_PREFIX = "backpacks.";
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java]: Also I still think this should be at the top, above this line `private final PlayerProfile profile;` with an empty line above it.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
Small mistake

**Review by TheBusyBiscuit — APPROVED** [review]
Thanks @b10n1k <3

**Review by poma123 — DISMISSED** [review]
Looking good!

**Review by poma123 — APPROVED** [review]

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**